code/modules/clothing/shoes/cowboy.dm

```diff
@@ -29,6 +29,10 @@
 				user.reagents.add_reagent(/datum/reagent/toxin, 7)
 		occupants.Cut()
 
+/obj/item/clothing/shoes/cowboy/Destroy()
+	occupants = null
+	..() //parent calls will take care of eliminating contents
```

Comment:
```suggestion
	return ..() //parent calls will take care of eliminating contents
```
without this we'll fail unit tests because it drops the qdel hint